Policies/KubernetesService/Azure_KubernetesService_AuthN_Enabled_AAD/Readme.md

```diff
@@ -0,0 +1,24 @@
+## Azure_KubernetesService_AuthN_Enabled_AAD
+
+### DisplayName 
+[Remove Azure Active Directory B2C tenant(s) in a subscription](../../../Control%20coverage/Feature/KubernetesService.md#Azure_KubernetesService_AuthN_Enabled_AAD)
```

Comment:
The control file name is for AKS - AAD control and display name is different.

Reply:
updated